### DISCUSSION TITLE
Run erigon without syncing

### DISCUSSION THREAD

**Discussion by lianqiao** [erigontech/erigon#3255] [Q&A] [answered]
Hi, 

I am running an erigon and want to dump transaction data from it. However, the erigon is running out of my 2TB SSD, only 30GB left. Can I run the erigon but disable syncing? So that I can use rpc API to get block data I want without running out of disk space.

One possible way is to disconnect network, but I still need to remotely access the computer.

Thanks!

**Comment by AskAlexSharov** [comment, marked as answer]
--nodiscover
